src/gsy_e/models/strategy/storage.py

```diff
@@ -51,11 +51,18 @@
 class StorageStrategy(BidEnabledStrategy):
     """Template strategy for storage assets."""
 
-    parameters = ("initial_soc", "min_allowed_soc", "battery_capacity_kWh",
-                  "max_abs_battery_power_kW", "cap_price_strategy", "initial_selling_rate",
-                  "final_selling_rate", "initial_buying_rate", "final_buying_rate", "fit_to_limit",
-                  "energy_rate_increase_per_update", "energy_rate_decrease_per_update",
-                  "update_interval", "initial_energy_origin", "balancing_energy_ratio")
+    def serialize(self):
+        return {
+            "initial_soc": self.state.initial_soc,
+            "min_allowed_soc": self.state.min_allowed_soc_ratio * 100.0,
+            "battery_capacity_kWh": self.state.capacity,
+            "max_abs_battery_power_kW": self.state.max_abs_battery_power_kW,
+            "cap_price_strategy": self.cap_price_strategy,
+            "initial_energy_origin": self.state._initial_energy_origin,
+            "balancing_energy_ratio": self.balancing_energy_ratio,
+            **self.bid_update.serialize(),
+            **self.offer_update.serialize(),
```

Comment:
interesting! With the [old way](https://github.com/gridsingularity/gsy-e/pull/1435/commits/4a3425118d3339ad3572aae03358d271b7865dfb#diff-5d95a36b16632bceaf86321630cbeb038104519dcd21aa330498a47d41914aabL89) of getting the parameters of a strategy class, these would not be part of  the parameters (this is what you meant by `but also dispatch the serialization to the offer/bid updater objects`, right?)

Reply:
Apparently this never was a problem. I wonder why we need the serialisation of the class parameters at all then. Can you please explain this? Thanks!

Reply:
This actually was a problem in the first place :D we just never used the serialized price values of the strategies that's why we never observed it. In the former implementation, no matter the values of `update_interval`, `initial_buying_rate`, `initial_selling_rate` ... serialization code never fetched these values correctly and always reported None for these. Admittedly we never used it, however the problem was there in the first place.